wiki_articles/windows_msys2_gcc.md

```diff
@@ -0,0 +1,35 @@
+# How to install msys2 and gcc on windows
+In this guide, we install msys2 and gcc. There are some optional tools that we recommend
+installing, such as `clangd` and `clang-format`. Having an alternative compiler, `clang`
+is also recommended. Install location is assumed to be the default.
+
+<!-- inline -->
+## Msys2 and GCC Toolchain
+1. Visit [msys2.org](https://msys2.org) and follow their installation guide upto step 4.
+2. After installing, open the Msys2 Msys terminal environment on windows. It is an app.
+3. Type `pacman -Syu` and press Enter to update.
+4. Reopen Msys2 Msys, and now we install the entire toolchain group.
+5. Type `pacman -S mingw-w64-ucrt-x86_64-toolchain`. This will install all the necessary tools.
+
+<!-- inline -->
+## Clang (Optional)
+1. Type `pacman -S mingw-w64-ucrt-x86_64-clang` and press enter.
+2. (Optional) Install clang tools, `pacman -S mingw-w64-ucrt-x86_64-clang-tools-extra`.
+
+## Add to path
+1. Click on Search and type "Edit Environment Variables". Select `Path` and click Edit.
+2. In the new window, click on New and then type `C:\msys64\ucrt64\bin` and click Ok.
+3. Open powershell and type `gcc -v` to check for correct output. If not, try logging back in.
+
+## Extras
+Make is installed as `mingw32-make.exe`.
```

Comment:
I think it'd be best to just install make at the same time as mingw, `pacman -S mingw-w64-ucrt-x86_64-gcc make gdb cmake` is what I usually do and it provides `make` instead of `mingw32-make`.  This might be simpler for beginners.

Reply:
Alright, will do.